### PR TITLE
Add parts for running Openshift plugin test nightly

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Adapt test to latest changes in UI of the plugin
* Add script for launching nightly on centos-ci
* Add method for selecting and getting text in terminal tabs with long names by prefix

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16927

### CODE REVIEW

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

**Comment by musienko-maxim** [comment]
[ci-test]

File: tests/.infra/centos-ci/functional_tests_utils.sh [hunk 1]
```diff
@@ -445,7 +445,7 @@ function runDevfileTestSuite() {
   -e TS_SELENIUM_LOAD_PAGE_TIMEOUT=240000 \
   -e TS_SELENIUM_WORKSPACE_STATUS_POLLING=20000 \
   -e NODE_TLS_REJECT_UNAUTHORIZED=0 \
-  quay.io/eclipse/che-e2e:nightly || IS_TESTS_FAILED=true
+  maxura/e2e-tests:CHE-16927 || IS_TESTS_FAILED=true
```
Comment (dmytro-ndp) [inline tests/.infra/centos-ci/functional_tests_utils.sh:449]: Please, don't forget to restore original address

File: tests/e2e/pageobjects/ide/Terminal.ts [hunk 2]
```diff
@@ -93,11 +94,22 @@ export class Terminal {
...
         const terminalIndex: number = await this.getTerminalIndex(terminalTab);
-        const terminalRowsXpathLocator: string = `(//div[contains(@class, 'terminal-container')]` +
-            `//div[contains(@class, 'terminal')]//div[contains(@class, 'xterm-rows')])[${terminalIndex}]`;
-
         await this.selectTerminalTab(terminalTab, timeout);
-        return await this.driverHelper.waitAndGetText(By.xpath(terminalRowsXpathLocator), timeout);
+        return await this.driverHelper.waitAndGetText(By.xpath(Terminal.TERMINAL_ROWS_XPATH_LOCATOR_PREFFIX +`[${terminalIndex}]`), timeout);
+    }
+
+    async selectTabByPreffixAndWaitText(terminalTab: string, expectedText: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/ide/Terminal.ts]: typo "Pre**ff**ix" > "Prefix"

File: tests/e2e/pageobjects/ide/Terminal.ts [hunk 3]
```diff
@@ -93,11 +94,22 @@ export class Terminal {
...
+        Logger.debug(`Terminal.selectTabByPreffixAndWaitText tab: ${terminalTab} text: ${expectedText}`);
+
+        const terminalTabLocatorWithPreffix: string = `//li[contains(@title, '${terminalTab}')]`;
+        const terminalIndex: number = await this.getTerminalIndex(terminalTab);
+        
+        await this.driverHelper.waitAndClick(By.xpath(terminalTabLocatorWithPreffix), timeout);
+        await this.driverHelper.waitUntilTrue(async () => {
+            const terminalText: string = await this.driverHelper.waitAndGetText(By.xpath(Terminal.TERMINAL_ROWS_XPATH_LOCATOR_PREFFIX +`[${terminalIndex}]`), timeout);
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/ide/Terminal.ts]: typo "PRE**FF**IX" > "PREFIX"

File: tests/e2e/TestConstants.ts [hunk 4]
```diff
@@ -238,12 +238,12 @@ export const TestConstants = {
     /**
      * Login for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
      */
-    TS_TEST_OPENSHIFT_PLUGIN_USERNAME: process.env.TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER || '',
+    TS_TEST_OPENSHIFT_PLUGIN_USERNAME: process.env.TS_TEST_OPENSHIFT_PLUGIN_USERNAME || '',
```
Comment (dmytro-ndp) [inline tests/e2e/TestConstants.ts:241]: What about usage of `TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER` like the following?:
> TS_TEST_OPENSHIFT_PLUGIN_USERNAME: process.env.TS_TEST_OPENSHIFT_PLUGIN_USERNAME || process.env.TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER || '',

File: tests/e2e/TestConstants.ts [hunk 5]
```diff
@@ -238,12 +238,12 @@ export const TestConstants = {
...
 
     /**
      * Password for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
      */
-    TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER || '',
+    TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_TEST_OPENSHIFT_PLUGIN_PASSWORD || '',
```
Comment (dmytro-ndp) [inline tests/e2e/TestConstants.ts:246]: What about usage of `TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER` like the following?:
> TS_TEST_OPENSHIFT_PLUGIN_PASSWORD: process.env.TS_TEST_OPENSHIFT_PLUGIN_PASSWORD || process.env.TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER || '',

Comment (dmytro-ndp) [reply tests/e2e/TestConstants.ts:246]: Sorry, I mislead `TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER` propose.
No need to change anything.

File: tests/e2e/TestConstants.ts (hunk 4 above)
Comment (dmytro-ndp) [reply tests/e2e/TestConstants.ts:241]: Sorry, I mislead `TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER` propose.
No need to change anything.

**Review by dmytro-ndp — APPROVED** [review]

**Comment by musienko-maxim** [comment]
[crw-ci-test]

**Comment by musienko-maxim** [comment]
[ci-build]

**Pull request merged**